### PR TITLE
Feedback Path: Predefined Combinations: Disable number field when not selected #8573

### LINKED ISSUE #8573: Feedback Path: Predefined Combinations: Disable number field when not selected
**Environment:**
`master` branch

**Steps to reproduce:**
- Login as instructor
- create a new session or edit a existing one
- click on `Add New Question`
- Select any option (e.g- essay)
- Under `Feedback Path` selected `Other Predefined Combinations...`
- Set any feedback path (e.g like in the screenshot below)
screenshot 394
- The number field in `The maximum number of *` is editable even when the radio button is not selected. (It is to be noted that setting the value without selecting the radio button doesn't save the changes)

**Expected behaviour:** The field should be disabled and only get enabled if the radio button is selected.

**Actual behaviour:** The field is enabled irrespective of which radio button is selected.

### PR DESCRIPTION
**Pull request by OlhaHn**
Fixes #8573

Some tests fail because the numbers field becomes disabled. Working on fixing that.

### CODE REVIEW

**Comment by wkurniawan07** [comment]
@OlhaHn it looks like you included the changes from #8581 here, which should not be the case.

**Comment by OlhaHn** [comment]
@wkurniawan07 yes, I see, my bad, I'll remove this changes in a few minutes.

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag [hunk 1]
```diff
@@ -93,16 +93,19 @@
       <div class="col-sm-8 form-control-static">
         <div class="col-sm-4 col-md-3 col-lg-2 margin-bottom-7px">
           <input class="nonDestructive" type="radio"
+              id="valueRatio-${fqForm.questionIndex}"
```
Comment (sukanta-27) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag]: Did you mean `valueRadio-${fqForm.questionIndex}`?

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag [hunk 2]
```diff
@@ -93,16 +93,19 @@
...
               value="custom" <c:if test="${!fqForm.editable}">disabled</c:if>>
           <input class="nonDestructive numberOfEntitiesBox width-75-pc" type="number"
+              id="numberOfTeams-${fqForm.questionIndex}"
               name="<%= Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFENTITIES %>"
               value="${fqForm.feedbackPathSettings.numOfEntitiesToGiveFeedbackToValue}"
               min="1" max="250" <c:if test="${!fqForm.editable}">disabled</c:if>>
         </div>
         <div class="col-sm-4 col-md-3 col-lg-2 margin-bottom-7px">
           <input class="nonDestructive" type="radio"
+              id="unlimitedRatio-${fqForm.questionIndex}"
```
Comment (sukanta-27) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag]: Similarly here. Did you mean `unlimitedRadio-${fqForm.questionIndex}`

**Comment by sukanta-27** [comment]
@OlhaHn Any updates here?

**Comment by jacoblipech** [comment]
Can I ask what is the issue with this PR that cannot be merged? I am wondering if I can help

**Comment by damithc** [comment]
> Can I ask what is the issue with this PR that cannot be merged? I am wondering if I can help

The first problem to fix is the CI failure. You may also want to sync the branch with upstream as the branch seems to be out-of-date.
image

**Comment by wkurniawan07** [comment]
@OlhaHn Closing due to inactivity for a long period. Feel free to request for reopen if you're ready to resume the work here.

**Pull request closed without merging**